apps/backend/migrations/20250320172703_tracks.js

```diff
@@ -0,0 +1,87 @@
+exports.up = async function (knex) {
+  await knex.schema.createTable('tracks', table => {
+    table.increments().primary()
+    table.string('name').notNullable()
+    table.text('description')
+    table.text('banner_url')
+    table.string('actions_name')
+    table.text('welcome_message')
+    table.text('completion_message')
+    table.string('completion_badge_name')
```

Comment:
So this isn't a direct reference to an existing group badge?

Reply:
you know, that would be easier and better. good call. i was just following aaron's designs